Cabal/doc/nix-local-build.rst

```diff
@@ -191,7 +191,7 @@ version of cabal-install:
    ``dist-newstyle/build/x86_64-linux/ghc-8.0.1/p-0.1/c/pexe/build/pexe/pexe``
    (you can see why we want this to be an implementation detail!)
 
-- In cabal-install-2.2 and above, the ``/c/`` part of the above path
```

Comment:
This led to some bad formatting

![Screenshot from 2020-06-30 14-17-05](https://user-images.githubusercontent.com/2716069/86133549-bf20d380-bae8-11ea-8b05-af13f48fb94b.png)